script.js

```diff
@@ -0,0 +1,132 @@
+// Variables
+let myLibrary = [];
+const bookShelf = document.getElementById('book-shelf');
+const modal = document.getElementById('modal');
+const modalShow = document.getElementById('show-modal');
+const modalClose = document.getElementById('close-modal');
+const addBookForm = document.getElementById('addbook-form');
+const bookTitleEl = document.getElementById('book-title');
+const bookAuthorEl = document.getElementById('book-author');
+const bookPriceEl = document.getElementById('book-price');
+const bookStatusEl = document.getElementById('book-status');
+
+// Book Class
+function Book(title, author, price, status) {
+  this.title = title;
+  this.author = author;
+  this.price = price;
+  this.status = status;
+}
```

Comment:
- [ ] The requirements specifically ask to implement a property `pages` for the Book object but a `price` property was implemented instead. We need to also add the `pages` property as requested.
![Screenshot from 2020-08-12 11-59-41](https://user-images.githubusercontent.com/30977845/90031228-5dc95400-dc93-11ea-863b-77e82f5580e2.png)